addons/sale_mrp/models/mrp_production.py

```diff
@@ -13,14 +13,18 @@ class MrpProduction(models.Model):
         groups='sales_team.group_sale_salesman')
     sale_line_id = fields.Many2one('sale.order.line', 'Origin sale order line')
 
-    @api.depends('procurement_group_id.mrp_production_ids.move_dest_ids.group_id.sale_id')
+    @api.depends('procurement_group_id.mrp_production_ids.move_dest_ids.group_id.sale_id', 'procurement_group_id.sale_id')
```

Comment:
`sale_line_id.order_id` should be in the depends too, no ?

Reply:
Thanks for the review, changes have been made and a test has been added